pallets/registries/src/tests.rs

```diff
@@ -1205,3 +1205,651 @@ fn update_registry_should_succeed() {
 		);
 	});
 }
+
+#[test]
+fn remove_delegate_should_fail_if_admin_authorization_is_not_found() {
+	let creator = ACCOUNT_00;
+	let delegate = ACCOUNT_01;
+	let registry = [2u8; 256].to_vec();
+
+	let raw_blob = [2u8; 256].to_vec();
+	let blob: RegistryBlobOf<Test> = BoundedVec::try_from(raw_blob)
+		.expect("Test blob should fit into the expected input length for the test runtime.");
+
+	let registry_digest = <Test as frame_system::Config>::Hashing::hash(&registry.encode()[..]);
+
+	let id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&registry_digest.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+
+	let registry_id: RegistryIdOf = generate_registry_id::<Test>(&id_digest);
+
+	let gibberish: Vec<u8> = vec![
+		42, 88, 173, 15, 94, 201, 120, 66, 33, 155, 77, 200, 11, 250, 92, 38, 63, 141, 222, 199,
+		81, 49, 7, 144, 99, 182, 241, 123, 56, 76, 190, 204,
+	];
+
+	let auth_id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&registry_id.encode()[..], &creator.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+
+	let gibberish_auth_id_digest = <Test as frame_system::Config>::Hashing::hash(&gibberish[..]);
```

Comment:
Can you do it like below?

```rust
let non_existent_auth_id_digest = <Test as frame_system::Config>::Hashing::hash(
		&[&registry_id.encode()[..]],
	);
```

Instead of using the bytes.